src/main/java/com/limechain/exception/misc/AuthorshipVerificationException.java

```diff
@@ -0,0 +1,7 @@
+package com.limechain.exception.misc;
+
+public class AuthorshipVerificationException extends RuntimeException {
```

Comment:
Where do we handle this type of exception?

Reply:
It's handled as a general `Exception` in `BlockHandler`. Since multiple exceptions can be thrown from methods in `handleBlockHeader()` I treat them equally as we currently handle a failure the same way from any kind of exception in that method.